src/kontrol/kdist/cheatcodes.md

```diff
@@ -907,6 +913,62 @@ Otherwise, throw an error for any other call to the Foundry contract.
       [owise]
 ```
 
+Mock calls
+----------
+
+#### `mockCall` - Mocks all calls to an address where if the call data either strictly or loosely matches data and returns retdata.
+
+```
+function mockCall(address where, bytes calldata data, bytes calldata retdata) external;
+```
+
+`foundry.call.mockCall` will match when the `mockCall` cheat code function is called.
+This rule then takes the `address` value from the function call data and etches a single byte into the account code, in case it is empty.
```

Comment:
It's a very minor thing, but I'd recommend to write "calldata" in one word to follow the Ethereum convention (here and below).

```suggestion
This rule then takes the `address` value from the function calldata and etches a single byte into the account code, in case it is empty.
```